browser/src/Config.ts

```diff
@@ -80,8 +80,8 @@ const DefaultPlatformConfig = Platform.isWindows() ? WindowsConfig : Platform.is
 
 Performance.mark("Config.load.start")
 
-const userConfigFile = path.join(getUserFolder(), "config.json")
-const userJsConfig = path.join(getUserFolder(), "config.js")
+export const userConfigFile = path.join(getUserFolder(), "config.json")
+export const userJsConfig = path.join(getUserFolder(), "config.js")
```

Comment:
Why do we have a `config.json` and a `config.js`?  Can we please come up with more unique filenames here?  I don't think the file extension should determine the difference between these two files.

Also, why is there a menu option to edit `config.js` but not one for editing `config.json`?  Maybe I'm missing something, I'm just confused.